app/views/user/contribute/_finished_tracks.erb

```diff
@@ -0,0 +1,3 @@
+<li>
+  <a href="/languages/<%= track['track_id'] %>/contribute" target='_blank'><%= track['language'] %></a>
```

Comment:
If we're going to have some logic to get the tracks, we may as well make the result an object:

```
track.id
track.language
```